docs/strat-lib/guides/howToSnipe.md

```diff
@@ -0,0 +1,54 @@
+---
+description: How to snipe an offer
+sidebar_position: 9
+
+---
+
+# Using `snipes` to take an offer
```

Comment:
I assume this is mostly the same as when inlined in the other doc so havn't read it in detail.
An issue with it for the next version of MGV is that at that point we can no longer snipe successful offers, only snipe for fail.
You could still use the cleaning functionality for testing but it would look more like a hack.

I think we shall leave it like this for now but make mental note that it requires future changes.

Reply:
Yep, it's exactly the same - just added a bit of wording here and there.

Noted for MGV V2 - I'll have to revisit every tutorial anyway, thanks for letting me know.

Reply:
(same, adapted for the WBTC/USDT pair that I used in the Smart offer tutorial, of course)

The sniping works, but it's not successful because of the funds pulling issue (router) we discussed.
I don't think it's a big deal anyway, since as you said, Sniping will only be for failing offers later